thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/route.js

```diff
@@ -423,6 +366,77 @@ export default Route.extend({
     }
   },
 
+  /**
+   * Fetches all anomaly data for found anomalies - downloads all 'pages' of data from server
+   * in order to handle sorting/filtering on the entire set locally. Start/end date are not used here.
+   * @param {Array} anomalyIds - list of all found anomaly ids
+   * @returns {RSVP promise}
+   */
+  fetchCombinedAnomalies: task(function * (anomalyIds) {
```

Comment:
From what I have researched, I can't yet use async/await here because they have not been integrated with concurrency due to "cancellability". Feel free to correct me if I'm wrong.